From cdf4ccd4f39d70e178070260de8e2c16b5bb1a5b Mon Sep 17 00:00:00 2001
From: "G.h. Gao" <guanhua.gao@nxp.com>
Date: Thu, 30 May 2019 15:20:25 +0800
Subject: [PATCH 182/235] drm/imx/hdp: Fix incompatible interface definitions
 introduced by backport

Fix incompatible interface definitions introduced by backport.

Signed-off-by: Guanhua Gao <guanhua.gao@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index 245052bb1b37..2b46c74926cf 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -15,7 +15,6 @@
 #include <linux/of.h>
 #include <linux/of_address.h>
 #include <linux/of_device.h>
-#include <drm/drm_probe_helper.h>
 
 #include "imx-hdp.h"
 #include "imx-hdmi.h"
@@ -756,8 +755,8 @@ static void imx_hdp_mode_setup(struct imx_hdp *hdp,
 }
 
 static void imx_hdp_bridge_mode_set(struct drm_bridge *bridge,
-				    const struct drm_display_mode *orig_mode,
-				    const struct drm_display_mode *mode)
+				    struct drm_display_mode *orig_mode,
+				    struct drm_display_mode *mode)
 {
 	struct imx_hdp *hdp = bridge->driver_private;
 
-- 
2.17.1

